org.tuxemon.Tuxemon.yaml

```diff
@@ -0,0 +1,52 @@
+---
+app-id: org.tuxemon.Tuxemon
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: org.tuxemon.Tuxemon
+rename-icon: icon
+finish-args:
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --filesystem=home
+- --share=network
+- --socket=pulseaudio
+- --device=dri
+modules:
+  # needed by pygame
+  # otherwise an error message occurs which complains about setuptools
```

Comment:
Maybe use https://github.com/flathub/shared-modules/blob/master/pygame/pygame-1.9.6.json instead as it contains PortMidi.

Reply:
I don't believe that we support pygame 1.9.6.  Its very old at this point and all testing is done on pygame versions newer thatn 2.0.0.  Tuxemon doesn't use midi, so if there is a build without portmidi, then that can be used.

Reply:
I will check it out.

Reply:
I did not find a way to disable midi